packit/utils.py

```diff
@@ -30,6 +30,7 @@ def get_rev_list_kwargs(opt_list):
     return result
 
 
+# TODO: we should use run_cmd from conu
```

Comment:
What's the problem with this? I do not want to depend on conu..;-)

Reply:
I would ideally copy the whole function from conu, not depend on it.

The one in conu is more powerful.

Reply:
If I remember correctly, I used that one but removed some parts, that I do not like, but forgot the reasons...;-( We can use the whole function from conu if you wish.

Reply:
I changed it, please take a look.